packages/widget-chat/src/actions/conversation.js

```diff
@@ -58,6 +66,9 @@ export function listenToMercuryActivity(conversationId, spark) {
     spark.mercury.on(`event:conversation.activity`, (event) => {
       const activity = event.data.activity;
       if (activity.object.objectType === `comment` && activity.target.id === conversationId) {
+        dispatch(receiveMercuryComment(activity));
+      }
+      else if (activity.object.objectType === `activity` && activity.target.id === conversationId) {
```

Comment:
Interesting, we hadn't encountered a lot of different types yet, so this will be good to add.